controller/src/controller/services/builder/prepuller.py

```diff
@@ -65,6 +66,16 @@ def build_pod(self, image: RSPImage, node: str) -> V1Pod:
                         name="prepull",
                         command=["/bin/true"],
                         image=image.reference_with_digest,
+                        resources=V1ResourceRequirements(
+                            limits={
+                                "cpu": "1",
+                                "memory": "512Mi",
+                            },
+                            requests={
+                                "cpu": "1m",
+                                "memory": "16Mi",
+                            },
+                        ),
```

Comment:
Let's set the limits to the same as the requests. Running `/bin/true` should never take that much memory.

Reply:
Also if you remove the trailing commas, ruff should  manage to make this a lot more compact.